pages/studentPortal/labs/multiplication/game.tsx

```diff
@@ -139,16 +140,16 @@ export default function BlockComponentGallery() {
                 ))
                 .reverse()}
             </div>
-            <div className="col-span-7 bg-rattata">
+            <div className="col-span-7 bg-gradient-to-r from-rattata ...">
```

Comment:
... ?

Reply:
![Screen Shot 2022-09-16 at 7 38 03 PM](https://user-images.githubusercontent.com/111075855/190687680-82209765-30f9-4ca3-853f-cd2f7c8d5bdc.png)
adds a fade to transparent gradient to the center of the game board